template.yaml

```diff
@@ -52,6 +52,10 @@ Parameters:
     Description: The name of the deployed IPV stub stack.
 
 Conditions:
+  UseCodeSigning:
```

Comment:
where is the UseCodeSigning used ?

Reply:
Removed UseCodeSigning from template.yaml as it's not used